cmd/wksctl/profile/enable/enable.go

```diff
@@ -0,0 +1,80 @@
+package enable
+
+import (
+	"errors"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/wksctl/cmd/wksctl/profile/constants"
+	"github.com/weaveworks/wksctl/pkg/git"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "enable",
+	Short: "Enable profile",
+	Args:  profileEnableArgs,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return profileEnableRun(profileEnableParams)
+	},
+}
+
+type profileEnableFlags struct {
+	gitUrl     string
+	revision   string
+	push       bool
+	profileDir string
+}
+
+var profileEnableParams profileEnableFlags
+
+func init() {
+	Cmd.Flags().StringVar(&profileEnableParams.profileDir, "profile-dir", "profiles", "specify a directory for storing profiles")
+	Cmd.Flags().StringVar(&profileEnableParams.gitUrl, "git-url", "", "enable profile from the gitUrl")
+	Cmd.Flags().StringVar(&profileEnableParams.revision, "revision", "master", "use this revision of the profile")
+	Cmd.Flags().BoolVar(&profileEnableParams.push, "push", true, "auto push after enable the profile")
+}
+
+func profileEnableArgs(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
+		return errors.New("profile enable does not require any argument")
+	}
+	return nil
+}
+
+func profileEnableRun(params profileEnableFlags) error {
+	repoUrl := params.gitUrl
+	if repoUrl == "" {
+		return errors.New("Git URL of the profile must be specified")
+	}
+
+	if repoUrl == constants.AppDevAlias {
+		repoUrl = constants.AppDevRepoURL
+	}
+
+	if git.IsGitURL(repoUrl) == false {
+		return errors.New("Repository is not a Git URL")
```

Comment:
```suggestion
		return errors.New("repository is not a Git URL")
```
it's idiomatic not to capitalize error strings :)

Reply:
per your another change, it is replaced by `return err`